### ISSUE TITLE
Playback data not being shown, possible cause: jellyfin-web not updating?

### ISSUE THREAD

**Issue by ikomhoog** [jellyfin/jellyfin#5357]
**Describe the bug**
When watching a video there is a button settings->playback data, and it should pop up a window with info about your stream(i think).
I don't actually know what info it is supposed to give since it doesn't work for me:
![image](https://user-images.githubusercontent.com/9083045/109783424-468b8d00-7c0a-11eb-900f-2d5b96d57dc4.png)
![image](https://user-images.githubusercontent.com/9083045/109784220-214b4e80-7c0b-11eb-8500-7fa3f9eaebf2.png)

My speculation(might be wrong):
I think this has something to do with jellyfin-web not being updated?
I was trying to mess around with quality settings earlier and moved the path of jellyfin-web.
I thought the container would download or extract the jellyfin-web things to that folder when it detected an empty folder but it didn't.
When would jellyfin-web be updated if I'm just running the base container?

**System (please complete the following information):**
 - OS: Unraid
 - Virtualization: Docker
 - Clients: Browser
 - Browser: Chrome
 - Jellyfin Version: 10.8.0 (unstable branch)
 - Playback: All
 - Installed Plugins: TMDb Box Sets
 - Reverse Proxy: nginx
 - Base URL: jellyfin.mydomain.com
 - Networking: Bridge
 - Storage: Local

**To Reproduce**

1. Watch a video
2. Click on Playback data
3. See the weird close button for the playback data window

**Expected behavior**
A window with playback data shown

**Logs**
There are no logs when I press the button

**Additional context**
I have been using jellyfin since before 10.4(i think), so if jellyfin-web doesn't update somehow it might be outdated and cause the issue.

**Comment by ikomhoog** [comment]
I don't know if I did it correctly but I somehow got here: https://dev.azure.com/jellyfin-project/jellyfin/_build/results?buildId=25861&view=artifacts&pathAsName=false&type=publishedArtifacts
And I downloaded the development version.
Once I copied that into my vs code jellyfin-web folder it worked.

So the problem is related to my jellyfin-web version, is there a way I can see what web version I'm using?
